src/CRTM_Forward_Module.f90

```diff
@@ -278,13 +278,13 @@ FUNCTION CRTM_Forward( &
        CALL SYSTEM_CLOCK (count_rate=count_rate)
        CALL SYSTEM_CLOCK (count=count_start)
     END IF
-    
+
     ! If no sensors or channels, simply return
     n_Sensors  = SIZE(ChannelInfo)
     n_Channels = SUM(CRTM_ChannelInfo_n_Channels(ChannelInfo))
     IF ( n_Sensors == 0 .OR. n_Channels == 0 ) RETURN
```

Comment:
An alternative would be to add a check here and return if `n_Profiles == 0`, but it would change the current behavior of the code, needing more testing. The less "invasive" would be what is in the PR already.